src/transformers/models/gpt2/configuration_gpt2.py

```diff
@@ -113,6 +113,12 @@ class GPT2Config(PretrainedConfig):
             Whether or not to use gradient checkpointing to save memory at the expense of slower backward pass.
         use_cache (:obj:`bool`, `optional`, defaults to :obj:`True`):
             Whether or not the model should return the last key/values attentions (not used by all models).
+        scale_attn_by_layer (:obj:`bool`, `optional`, defaults to :obj:`False):
```

Comment:
```suggestion
        scale_attn_by_layer_id (:obj:`bool`, `optional`, defaults to :obj:`False):
```
It's really scaled by the layer_id no? Think that's a tiny bit clearer

Reply:
`scale_attn_by_layer_idx` is now the official name!